### ISSUE TITLE
Openelec 5.0.8 Operative system not supported!

### ISSUE THREAD

**Issue by sssgsm** [i96751414/anonymous-repo#9]
Hi, I'm using openelec 5.0.8 on raspberry pi 2 and k0di tools 1.1.6

When I execute k0di tools it give me a message "Operative system not supported!"

I'd tried activating option "force openelec" and "openelec lib->librtmp.so.1" without results.

Any idea/trick?

Thanks

**Comment by i96751414** [comment]
No... At the time i don't have any way to solve openelec issues, because i can't test.
